modules/schedule_module/test/TestPlan.md

```diff
@@ -0,0 +1,33 @@
+# Schedule Module Test Plan
+
+1. Create an Appointment [Automation Testing]
+Ensure that create does NOT work if one or more of the fields are empty. 
```

Comment:
I am able to create an appointment without the time field filled as it's not a required field. is this intentional? if so, this test plan needs to be updated